core/tx_pool.go

```diff
@@ -630,6 +646,14 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 	if tx.Gas() < intrGas {
 		return ErrIntrinsicGas
 	}
+
+	// Check if the sender account is authorized to perform the transaction
+	if isQuorum {
+		if err := checkAccount(from, tx.To()); err != nil {
+			return ErrUnAuthorizedAccount
```

Comment:
User experience: It'd be good to append the message from `err` object to the final error.

Reply:
changed.